tools/data-handler/src/calculate.ts

```diff
@@ -402,35 +331,55 @@ fieldtype(X, Field, "cardkeys") :- field(X, Field, _), card(Value) : field(X, Fi
   public async run(
     projectPath: string,
     cardKey: string,
-  ): Promise<ParseResult[] | undefined> {
+  ): Promise<ParseResult | undefined> {
     Calculate.project = new Project(projectPath);
 
-    const card = await Calculate.project.findSpecificCard(cardKey);
+    /*     const card = await Calculate.project.findSpecificCard(cardKey);
```

Comment:
Should the commented block be removed? If not, add reason why the block is commented out.

Reply:
This is still a draft PR and the run method was temporarily changed so that you can try running query files